jdaviz/app.py

```diff
@@ -327,6 +327,11 @@ def load_data(self, file_obj, parser_reference=None, **kwargs):
             snackbar_message = SnackbarMessage("Data successfully loaded.",
                                                sender=self)
             self.hub.broadcast(snackbar_message)
+        except Exception:  # Reset state on uncaught errors
+            cfg_name = self.state.settings.get('configuration', 'unknown')
+            if cfg_name in ('mosviz', ):  # Add more as needed.
```

Comment:
This keeps the fix in upper-level app code but allow us to pick and choose which viz get that fix.